src/utils/xhr-loader.ts

```diff
@@ -100,15 +100,15 @@ class XhrLoader implements Loader<LoaderContext> {
     if (xhrSetup) {
       Promise.resolve()
         .then(() => {
-          if (this.stats.aborted) return;
+          if (this.stats?.aborted) return;
```

Comment:
We want to return early here if the loader was destroyed (stats is null) or if the request was aborted. So

```suggestion
          if (this.stats === null || this.stats.aborted) return;
```